### PR TITLE
Blog app: add devise

### PR DESCRIPTION
**Pull request by fatima-najafi**
# Hello and welcome to this PR! 👋🏻 
### Blog app: add devise
[Project requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_devise.md#project-requirements)
Remove current_user method in ApplicationController, devise will provide us with one.
[Install and setup devise](https://github.com/heartcombo/devise#getting-started).
Can register a new user.
User logs in with a combination of email and password.
Hashed password  stored in the database.
Ask for confirmation of email.
Can reset password.
[Modify the views in devise](https://github.com/heartcombo/devise#configuring-views) for registration and login to match [the wireframes from the Sneak Peek](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md#:~:text=For%20this%20project%20you%20will%20have%20full%20freedom%20in%20terms%20of%20visual%20design%20but%20you%20will%20need%20to%20keep%20the%20following%20wireframes%3A) and your styling.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,12 +1,15 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```
Comment (uwadonat) [inline app/models/user.rb]: - [x] Let's make sure that the application asks for confirmation of email.
 `:confirmable`

**Review by uwadonat — CHANGES_REQUESTED** [review]
## Hi @Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Your commit messages are very descriptive ✔️
- Correct work-flow was used ✔️
- No linter errors ✔️
- Detailed pull request message ✔️
almost

## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/user.rb (hunk 1 above)
Comment (uwadonat) [reply app/models/user.rb]: Hello @Team. By mistake I approved your project wich has a change to be performed. Kindly fix the change requested above before merging this pull request. you need to add `:confirmable` in your user model.

your code will look as bellow:
````ruby
devise :database_authenticatable, :registerable,
				:recoverable, :rememberable, :validatable, :confirmable
````

after correcting this, you can merge your pull request and proceed with the next project. no need to submit for another review on this pull request.

Thank you.

Comment (fatima-najafi) [reply app/models/user.rb]: Thank you  very much @uwadonat  . I made the changes

**Pull request merged**